articles/article-060-handshake.md

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Introductions are universal
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In every society there is some sort of formal introduction when people meet each other for the first time. This is such a fundamental process that it even finds application in computers.
+Usually when different processes communicate with each other there also exists this introduction phase at the beginning. In computer science we call this the handshake. This is the time, where the different processes agree to communicate with each other and clearly state their protocol. Sometimes the handshake is also used to identify each other by means of private keys or passwords.
```

Comment:
Is the sentence "Usually, when different process ...." needed? I find it better without it. What do you think?

Reply:
I agree. I removed the sentence and restructured the sentence.